dotCMS/src/main/java/com/dotmarketing/factories/DBTreeTransformer.java

```diff
@@ -0,0 +1,46 @@
+package com.dotmarketing.factories;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.dotmarketing.beans.Tree;
+import com.google.common.collect.ImmutableList;
+
+public class DBTreeTransformer {
+	private final List<Tree> _trees;
+	
+	public DBTreeTransformer(final List<Map<String, Object>> trees){
+		this._trees = listToTreeList(trees);
+	}
+	public DBTreeTransformer(final Map<String, Object> tree){
+		this._trees = ImmutableList.of(mapToTree(tree));
+	}
+	
+	public List<Tree> trees(){
+		return _trees;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "_trees". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)